HW04.Rmd

```diff
@@ -1,126 +1,266 @@
+
+### Are whites who are killed by guns more likely to die because of suicide or homicide? How does this compare to blacks and hispanics?
+
+```{r race}
+#for white
+white_gun <- subset(gun, race == "White")
+white_gun <- white_gun["intent"]
+white_gun <- as.data.frame(table(white_gun))
+colnames(white_gun) <- c("intent", "death")
+
+suicide <- which(white_gun$intent == "Suicide")
+homicide <- which(white_gun$intent == "Homicide")
+
+white_gun[suicide, 2] > white_gun[homicide, 2]
+
+
+#for black
+black_gun <- subset(gun, race == "Black")
+black_gun <- black_gun["intent"]
+black_gun <- as.data.frame(table(black_gun))
+colnames(black_gun) <- c("intent", "death")
+
+suicide <- which(black_gun$intent == "Suicide")
+homicide <- which(black_gun$intent == "Homicide")
+
+black_gun[suicide, 2] > black_gun[homicide, 2]
+
+
+#for hispanics
+hispanic_gun <- subset(gun, race == "Hispanic")
+hispanic_gun <- hispanic_gun["intent"]
+hispanic_gun <- as.data.frame(table(hispanic_gun))
+colnames(hispanic_gun) <- c("intent", "death")
+
+suicide <- which(hispanic_gun$intent == "Suicide")
+homicide <- which(hispanic_gun$intent == "Homicide")
+
+hispanic_gun[suicide, 2] > hispanic_gun[homicide, 2]
+
+```
+
+(hint maybe looking at percentages would be good)
+```{r race_2}
+
+```
+
+
+### Are police-involved gun deaths significantly different from other gun deaths? Assess the relationship between police involvement and age, police involvement and race, and the intersection of all three variables.
+
+```{r police}
+gun_police_age_race <- gun[,c("police", "race", "age")]
+gun_police_age_race <- subset(gun_police_age_race, age != "Not Available")
+
+
+#police involvement and age
+gun_police_age <- gun_police_age_race[,c("police", "age")]
+gun_police_age_1 <- subset(gun_police_age, police == 1)
+#mean age with police involvement
+mean(as.numeric(gun_police_age_1[,"age"]))
+
+gun_police_age_0 <- subset(gun_police_age, police == 0)
+#mean age without police involvement
+mean(as.numeric(gun_police_age_0[,"age"]))
+
+
+
+#police involvement and race
+gun_police_race <- gun_police_age_race[,c("police", "race")]
+gun_police_race <- as.data.frame(table(gun_police_race))
+#get percentage of police involvement for each race can solve the problem
+
+which(gun_police_race$race == "Asian/Pacific Islander")
+which(gun_police_race$race == "Black")
+which(gun_police_race$race == "Hispanic")
+which(gun_police_race$race == "Not Availabletive American/Not Availabletive Alaskan")
+which(gun_police_race$race == "White")
+
+
+
+```
+
+Think about performing stats such as a t.test to see if differences you are plotting are statistically significant
+
+```{r stats?}
+
+
+```
+
+
+```{r police_2}
+
+```
+
+```{r stats_2?}
+
+
+```
+
+
+Finally, all together now:
+```{r police_3}
+
+```
+
+
+## Session info
+
+```{r}
+# always good to have this for reproducibility purposes
+devtools::session_info()
+```
+
```

Comment:
Great job! Sorry I could not give much critique, I am just learning :) I did learn a lot from reading your code and searching some things I was unfamiliar with!